### ISSUE TITLE
The parameter LinuxFxVersion has an invalid value

### ISSUE THREAD

**Issue by MisinformedDNA** [microsoft/azure-pipelines-tasks#11686]
**Question, Bug, or Feature?**  
*Type*: Bug

**Enter Task Name**: AzureRmWebAppDeployment@4

## Environment
- Server - Azure Pipelines
    
- Agent - Hosted 
    
## Issue Description

I used the YAML pipeline assistant to create this task:

```yaml
- task: AzureRmWebAppDeployment@4
  inputs:
    ConnectionType: 'AzureRM'
    azureSubscription: 'XXXX'
    appType: 'webAppLinux'
    WebAppName: 'XXXX'
    packageForLinux: '$(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip'
    RuntimeStack: 'DOTNETCORE|3.0'
    enableCustomDeployment: true
    DeploymentType: 'zipDeploy'
```

But when it runs, I get the following error:

> Error: Failed to patch App Service 'XXXX' configuration. Error: BadRequest - The parameter LinuxFxVersion has an invalid value. (CODE: 400)

The code is being deployed to a Web App (.NET Core 3.0 stack) on a Linux App Service Plan.

### Task logs

[logs_71.zip](https://github.com/microsoft/azure-pipelines-tasks/files/3804145/logs_71.zip)

### Error logs

> ##[debug][PATCH]https://management.azure.com/subscriptions/XXXXXXXX-7c9b-4726-9bdb-8dea951e5b6b/resourceGroups/XXXX/providers/Microsoft.Web/sites/XXXX/config/web?api-version=2016-08-01
> ##[debug]Deployment Failed with Error: Error: Failed to patch App Service 'XXXX' configuration. Error: BadRequest - The parameter LinuxFxVersion has an invalid value. (CODE: 400)
> ##[debug]task result: Failed
> ##[error]Error: Failed to patch App Service '' configuration. Error: BadRequest - The parameter LinuxFxVersion has an invalid value. (CODE: 400)
> ##[debug]Processed: ##vso[task.issue type=error;]Error: Failed to patch App Service 'XXXX' configuration. Error: BadRequest - The parameter LinuxFxVersion has an invalid value. (CODE: 400)
> ##[debug]Processed: ##vso[task.complete result=Failed;]Error: Failed to patch App Service 'XXXX' configuration. Error: BadRequest - The parameter LinuxFxVersion has an invalid value. (CODE: 400)

**Comment by gpalsson** [comment]
Yes I have same problem. Any fix for it yet?

**Comment by juanluelguerre** [comment]
Same problem for me running  .NET Core 2.2 app.
I appreciate any help about it !
Thanks a lot

**Comment by eaarora-ms** [comment]
@juanluelguerre  Can you please share your debug logs with system.debug set as true?
@MisinformedDNA Can you please confirm is it's still happening? We are assuming it might be because of API failure since I am not able to repro the issue at my end.

**Comment by juanluelguerre** [comment]
Hey @eaarora-ms ,
In my case, everything works properly after create a new Azure App Service (Containers) from scratch. 

Thanks.

**Comment by eaarora-ms** [comment]
Great! Looks like it was because of API failure. Waiting on customer response to close the issue.

**Comment by eaarora-ms** [comment]
Please comment to re-open the issue if you see it again.

**Comment by MisinformedDNA** [comment]
You should create a new issue.
